Adjusting "Shirakami Fubuki" title.

bempfoobq052021 opened the pull request:
Changes in custom.css : .presentation-title, .footer-nav line-height (accomodating mobile version).
Changes in index.html : 
```
<h1> to <span> (accomodating mobile version) 
style=font moved to custom.css
line-height added to adjust <span> to look more natural like <h1>
```

bempfoobq052021 committed:
Update custom.css

bempfoobq052021 committed:
Merge pull request #1 from bempfoobq052021/bempfoobq052021-patch-footer-nav-footer-credit-80to16pt

Update on custom css : .footer-nav, .footer .credits { line-height: 85px = 20px; }

bempfoobq052021 committed:
Update custom.css

Additional background to custom class .presentation-title to control the color, -webkit-bg-clip to text, -webkit-txt-fc to make the text transparent

bempfoobq052021 committed:
Update index.html

Cosmetic changes.
Changing from <h1> tag to <span> tag because h1 functions like <p> and handles color gradient per whole sentence rather than per word. <span> handles color gradient per word.

bempfoobq052021 committed:
Update index.html

bempfoobq052021 committed:
Changes in main backgroudn image

Custom.css
<video> min width, width, and left align
.presentation-title adjust for responsive mobile

index.html
changes inline style to css based style.

nevrending commented on assets/css/custom.css:
```diff
@@ -15,12 +15,32 @@ video {
 
+@media screen and (max-width: 768px) {
+  video {
+	min-width: 320px;
```
```suggestion
    min-width: 320px;
```

nevrending commented on assets/css/custom.css:
```diff
@@ -15,12 +15,32 @@ video {
+	background: linear-gradient(180deg,#fff 35%,#8fb2c3);
+	-webkit-background-clip: text;
+	-webkit-text-fill-color: transparent;
+	line-height: 6rem;
```
```suggestion
    font-size: 5.5rem;
    background: linear-gradient(180deg,#fff 35%,#8fb2c3);
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent;
    line-height: 6rem;
```

bempfoobq052021 replied:
Looks good to me~!

nevrending commented on assets/css/custom.css:
```diff
@@ -15,12 +15,32 @@ video {
+
+@media screen and (max-width: 767px) {
+  .presentation-title {
+	font-size: 3.5rem;
+	line-height: 5rem;
```
```suggestion
    font-size: 3.5rem;
    line-height: 5rem;
```

bempfoobq052021 replied:
Looks good to me~! Thanks for the fix!

nevrending commented on index.html:
```diff
@@ -71,7 +71,7 @@
       <div class="container">
         <div class="title-brand" style="max-width: 960px">
-          <h1 class="presentation-title" style="font-size: 7em">Shirakami Fubuki</h1>
+          <span class="presentation-title">Shirakami Fubuki</h1>
```
Tag has not been closed properly.

```suggestion
          <span class="presentation-title">Shirakami Fubuki</span>
```

bempfoobq052021 replied:
Whoops. I'll accept this immediately.

nevrending reviewed (changes requested):
Hi there! Thanks for your PR. It makes the mobile view looks better.

![image](https://user-images.githubusercontent.com/4745298/117479370-99374b80-afa3-11eb-9602-beab4f922820.png)

Just a couple of things:
- We use space for indentation instead of tabs
- The `span` tag that replaced `h1` hasn't been closed
- Footer CSS fixed mobile, but makes desktop looks misaligned (see below).

![image](https://user-images.githubusercontent.com/4745298/117479582-d8659c80-afa3-11eb-9781-744489846c19.png)

I've put in some inline comments that you can directly commit or you can make a commit that contains all of them, whichever works best for you.

bempfoobq052021 committed:
Update index.html

<span> : </h1> to </span>

Co-authored-by: Yefta Sutanto <yeftasutanto@gmail.com>

bempfoobq052021 committed:
Update assets/css/custom.css

space changes.

Co-authored-by: Yefta Sutanto <yeftasutanto@gmail.com>

bempfoobq052021 committed:
Update assets/css/custom.css

space changes.

Co-authored-by: Yefta Sutanto <yeftasutanto@gmail.com>

bempfoobq052021 committed:
Update assets/css/custom.css

space changes.

Co-authored-by: Yefta Sutanto <yeftasutanto@gmail.com>

bempfoobq052021 commented:
On this problem : 
Footer CSS fixed mobile, but makes desktop looks misaligned (see below).

Fixed by changing `<span class="copyright"> to <p class="copyright" style="line-height:20pt">`
I use &nbsp for the spaces though.

bempfoobq052021 committed:
Footer copyright alignment

Custom.css changes in flex align-items: center etc..
index.html changes in <span class="copyright"> to <p class=....>

bempfoobq052021 committed:
Merge branch 'master' of https://github.com/bempfoobq052021/foob

nevrending commented on index.html:
```diff
@@ -233,18 +233,16 @@ <h2 class="title">Community</h2>
   <span id="sideimg-left"></span>
   <span id="sideimg-right"></span>
   <script src="assets/js/core/jquery.min.js"></script>
-  <script src="assets/js/core/popper.min.js"></script>
-  <script src="assets/js/core/bootstrap.min.js"></script>
-  <script src="assets/js/theme.min.js?v=2.2.0"></script>
```
These should not be removed as they enabled the dropdown menu and stuffs.

nevrending reviewed (approved):
I'll accept this PR for now, since the hero fix is good, and work on the remaining issues on a separate commit.

Thanks again for your contribution!
